src/test/java/org/cactoos/iterable/JoinedTest.java

```diff
@@ -75,25 +73,26 @@ public void joinsIterablesWithSize() {
             new IterableOf<>(1, -1, 0, 0),
             new IterableOf<>(1)
         );
-        MatcherAssert.assertThat(
-            "Can't concatenate four iterables together",
-            list, Matchers.iterableWithSize(10)
-        );
-        MatcherAssert.assertThat(
-            "Can't concatenate four iterables together, again",
-            list, Matchers.iterableWithSize(10)
-        );
+        new Assertion<>(
+            "Must concatenate four iterables together",
+            list, new HasSize(10)
+        ).affirm();
+        new Assertion<>(
+            "Must concatenate four iterables together, again",
+            list, new HasSize(10)
+        ).affirm();
     }
 
     @Test
     public void joinItemAndIterable() {
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Must join item and iterable",
             new Joined<>(
                 0,
                 new IterableOf<>(1, 2, 3)
             ),
-            Matchers.contains(0, 1, 2, 3)
-        );
+            new HasValues<>(0, 1, 2, 3)
```

Comment:
@victornoel Here, we should assert for the whole expected iterable, not if it just have the values